### PR TITLE
[WIP] Language review

### PR DESCRIPTION
**Pull request by sephcoster**
Short description explaining the high-level reason for the pull request

## Additions

-

## Removals

-

## Changes

-

## Testing

-

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/components/Trends/TrendsPanel.jsx [hunk 1]
```diff
@@ -38,16 +39,34 @@ const subLensMap = {
   product: 'Products'
 }
 
+const lensHelperTextMap = {
+  'product': 'Product the consumer identified in the complaint.' +
```
Comment (flacoman91) [inline src/components/Trends/TrendsPanel.jsx:43]: 👍   i think eslint will nag you for the quote around the keys here though.

File: src/components/Trends/TrendsPanel.jsx [hunk 2]
```diff
@@ -59,6 +78,9 @@ export class TrendsPanel extends React.Component {
   }
 
   _phaseMap() {
+    const minDate = moment(this.props.date_received_min).format('MM/DD/YYYY')
```
Comment (flacoman91) [inline src/components/Trends/TrendsPanel.jsx:81]: I would move these down to the mapStatetoProps so you can just do this.props.minDate

File: src/reducers/trends.jsx [hunk 3]
```diff
@@ -135,6 +135,20 @@ function getD3Names( obj, nameMap, expandedTrends ) {
   }
 }
 
+/**
+ * helper function to pluralize field values
+ * @param {lens} lens value we are processing
+ * @returns {string} for consumption by AreaData function
+ */
+export function mainNameLens( lens ) {
```
Comment (flacoman91) [inline src/reducers/trends.jsx:143]: 👍

File: src/reducers/trends.jsx [hunk 4]
```diff
@@ -301,7 +317,12 @@ export const getColorScheme = ( lens, rowNames ) => {
   }
 
   colScheme.Complaints = colors.BriteCharts.medium
-  colScheme.Other = colors.DataLens[10]
+
+  // Set constant grey colors for our "other" buckets"
+  // TODO: Set these as constants / consolidate colors across charts
+  colScheme['All other products'] = colors.DataLens[10]
```
Comment (flacoman91) [inline src/reducers/trends.jsx:323]: 👍

File: src/components/Trends/TrendsPanel.jsx (hunk 1 above)
Comment (sephcoster) [reply src/components/Trends/TrendsPanel.jsx:43]: It first nagged me about the quotes, but then also nagged about consistency for using quotes and not using quotes. I think the latter rule trumped the former but I'll ensure the nag is gone either way.

File: src/reducers/query.jsx [hunk 5]
```diff
@@ -757,11 +760,14 @@ function changeFocus( state, action ) {
...
+  const { lens } = action
+
   return {
     ...state,
     focus: '',
-    lens: action.lens,
-    subLens: getSubLens( action.lens )
+    lens,
+    subLens: getSubLens( lens ),
+    trendDepth: lens === 'Company' ? 10 : 5
```
Comment (flacoman91) [inline src/reducers/query.jsx:770]: 👍  i forgot about this one

File: src/components/Charts/RowChart.jsx [hunk 6]
```diff
@@ -151,6 +151,7 @@ export class RowChart extends React.Component {
       this.props.total > 0 &&
       <div className="row-chart-section">
         <h3>{ this.props.title }</h3>
+        <p>{ this.props.helperText }</p>
```
Comment (flacoman91) [inline src/components/Charts/RowChart.jsx:154]: you can add helperText to the RowChart.propTypes and make it optional if you want.  Not really necessary, but we have that in this component.

**Comment by sephcoster** [comment]
Thanks for the feedback. Closing in favor of another approach to "All Other"

**Pull request closed without merging**